crates/bevy_remote/src/builtin_methods.rs

```diff
@@ -1491,6 +1733,30 @@ fn get_component_type_registration<'r>(
         .ok_or_else(|| anyhow!("Unknown component type: `{}`", component_path))
 }
 
+/// Given a resource's type path, return the associated [`ReflectResource`] from the given
+/// `type_registry` if possible.
+fn get_reflect_resource<'r>(
+    type_registry: &'r TypeRegistry,
+    resource_path: &str,
+) -> AnyhowResult<&'r ReflectResource> {
+    let resource_registration = get_resource_type_registration(type_registry, resource_path)?;
+
+    resource_registration
+        .data::<ReflectResource>()
+        .ok_or_else(|| anyhow!("Resource `{}` isn't reflectable", resource_path))
+}
+
+/// Given a resource's type path, return the associated [`TypeRegistration`] from the given
+/// `type_registry` if possible.
+fn get_resource_type_registration<'r>(
```

Comment:
Really minor: this could be imho named `get_type_registration` since it would work the same way for the non resource type. In case that is changed, change also the `resource_path` argument name.